src/app/modules/item/services/item-task.service.ts

```diff
@@ -68,12 +70,16 @@ export class ItemTaskService {
   }
 
   private bindPlatform(task: Task): void {
+    if (!this.attemptId) throw new Error('attemptId must be defined');
+    const randomSeed = Number(this.attemptId);
```

Comment:
The reason ids are string is that they are big int, so they are up to 2^63-1 while JS numbers loose the precision when >2^52. So mapping an id from the backend to JS may end up with many collisions in the high numbers, which is pretty annoying for a random seed (doing +1 would give the same seed).
But at the same time, currently attempt ids are incremental (they are the ones) and in such a case, it becomes a problem only we have >2^52 attempts... which is pretty safe probably.
Possibly, it is worth adding a comment such as `// attempt id can be used as a seed as these are currently assigned incrementally by participant id. If this changes, this needs to be adapted. `

Reply:
fixed